server/issue.go

```diff
@@ -647,19 +648,18 @@ func (p *Plugin) assignJiraIssue(mmUserId, issueKey, assignee string) (string, e
 		return errorMsg, nil
 	}
 
-	// Get list of assignable assignees
-	url := fmt.Sprintf("rest/api/2/user/assignable/search?issueKey=%s&username=%s", issueKey, assignee)
-	req, err := jiraClient.NewRequest("GET", url, nil)
-	if err != nil {
-		return "", err
+	// Get list of assignable users
+	var jiraUsers []jira.User
+	status, err := JiraGet(jiraClient, "user/assignable/search",
```

Comment:
:+1: for `JiraGet`

Reply:
Note, my linter warns me of dead code. Should we hook that sort of linting into CI?